virttest/utils_misc.py

```diff
@@ -3469,6 +3469,34 @@ def check_device_driver(pci_id, driver_type):
     return driver == driver_type
 
 
+def get_grub_file(session=None):
```

Comment:
get_bootloader_cfg looks better, it looks bootloader in power platfrom is not grub.